### PR TITLE
feat(components/ag-grid): support AG Grid 32.2.2

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#3105450](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/3105450)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2816/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/a11y-storybook/)
- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/colorpicker-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/core-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/forms-storybook/)
- [help-inline-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/help-inline-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/modals-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/popovers-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/2816/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2816/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2816/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2816/playground/)

File: libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts [hunk 1]
```diff
@@ -532,10 +523,10 @@ export class SkyAgGridService implements OnDestroy {
       },
       loadingOverlayComponent: SkyAgGridLoadingComponent,
       onCellFocused: () => this.#onCellFocused(),
-      selection: {
+      rowSelection: {
```
Comment (johnhwhite) [inline libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts:526]: They renamed `selection` back to `rowSelection`.

File: libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts [hunk 2]
```diff
@@ -532,10 +523,10 @@ export class SkyAgGridService implements OnDestroy {
...
         mode: 'multiRow',
         enableClickSelection: false,
-        enableMultiSelectWithClick: true,
+        enableSelectionWithoutKeys: true,
```
Comment (johnhwhite) [inline libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts:529]: This was renamed.

File: libs/components/ag-grid/src/lib/modules/ag-grid/fixtures/ag-grid-data-manager.component.fixture.ts [hunk 3]
```diff
@@ -83,7 +83,7 @@ export class SkyAgGridDataManagerFixtureComponent implements OnInit {
   }
 
   public ngOnInit(): void {
-    this.gridOptions = this.#gridService.getEditableGridOptions({
+    this.gridOptions = this.#gridService.getGridOptions({
```
Comment (johnhwhite) [inline libs/components/ag-grid/src/lib/modules/ag-grid/fixtures/ag-grid-data-manager.component.fixture.ts:86]: This probably always should've been `getGridOptions` — we remove `rowSelection` in `getEditableGridOptions`, and it now does nothing when calling `selectAll` when `rowSelection` is not configured.

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Pull request merged**